front-end/src/services/teachersService.js

```diff
@@ -0,0 +1,6 @@
+import API from 'API';
+
+export const getTeachers = async () => {
```

Comment:
What is difference between `teachersService.js` and `teacherService`?

Reply:
because of inattention did not see that